<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="com-PC" name="com.instaone.datadriven.Customer_DataDriven" tests="12" failures="0" timestamp="01 Jun 2020 09:44:46 GMT" time="123.011" errors="1">
  <testcase name="Login" time="29.022" classname="com.instaone.datadriven.Customer_DataDriven"/>
  <testcase name="first" time="22.815" classname="com.instaone.datadriven.Customer_DataDriven"/>
  <testcase name="first" time="8.596" classname="com.instaone.datadriven.Customer_DataDriven"/>
  <testcase name="first" time="5.167" classname="com.instaone.datadriven.Customer_DataDriven"/>
  <testcase name="first" time="7.213" classname="com.instaone.datadriven.Customer_DataDriven"/>
  <testcase name="first" time="3.640" classname="com.instaone.datadriven.Customer_DataDriven"/>
  <testcase name="first" time="3.778" classname="com.instaone.datadriven.Customer_DataDriven">
    <error type="org.openqa.selenium.UnhandledAlertException" message="unexpected alert open: {Alert text : Mobile No is already registered.}
  (Session info: chrome=83.0.4103.61): Mobile No is already registered.
Build info: version: &#039;3.141.59&#039;, revision: &#039;e82be7d358&#039;, time: &#039;2018-11-14T08:17:03&#039;
System info: host: &#039;COM-PC&#039;, ip: &#039;192.168.43.34&#039;, os.name: &#039;Windows 7&#039;, os.arch: &#039;x86&#039;, os.version: &#039;6.1&#039;, java.version: &#039;1.8.0_171&#039;
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 83.0.4103.61, chrome: {chromedriverVersion: 83.0.4103.39 (ccbf011cb2d2b..., userDataDir: C:\Users\com\AppData\Local\...}, goog:chromeOptions: {debuggerAddress: localhost:51110}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:virtualAuthenticators: true}
Session ID: cc1c68df40fc716193ff6ea224286154
*** Element info: {Using=id, value=txtMobile2}">
      <![CDATA[org.openqa.selenium.UnhandledAlertException: unexpected alert open: {Alert text : Mobile No is already registered.}
  (Session info: chrome=83.0.4103.61): Mobile No is already registered.
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'COM-PC', ip: '192.168.43.34', os.name: 'Windows 7', os.arch: 'x86', os.version: '6.1', java.version: '1.8.0_171'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 83.0.4103.61, chrome: {chromedriverVersion: 83.0.4103.39 (ccbf011cb2d2b..., userDataDir: C:\Users\com\AppData\Local\...}, goog:chromeOptions: {debuggerAddress: localhost:51110}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:virtualAuthenticators: true}
Session ID: cc1c68df40fc716193ff6ea224286154
*** Element info: {Using=id, value=txtMobile2}
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:120)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:49)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:158)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:323)
at org.openqa.selenium.remote.RemoteWebDriver.findElementById(RemoteWebDriver.java:372)
at org.openqa.selenium.By$ById.findElement(By.java:188)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:315)
at org.openqa.selenium.support.pagefactory.DefaultElementLocator.findElement(DefaultElementLocator.java:69)
at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:38)
at com.sun.proxy.$Proxy10.toString(Unknown Source)
at java.lang.String.valueOf(String.java:2994)
at java.lang.StringBuilder.append(StringBuilder.java:131)
at com.instaone.generic.BasePage.verifyElementPresent(BasePage.java:64)
at com.instaone.pompages.Customer.Enter_CustomerPhone(Customer.java:72)
at com.instaone.datadriven.Customer_DataDriven.first(Customer_DataDriven.java:67)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- first -->
  <testcase name="first" time="4.095" classname="com.instaone.datadriven.Customer_DataDriven"/>
  <testcase name="first" time="7.615" classname="com.instaone.datadriven.Customer_DataDriven"/>
  <testcase name="first" time="4.674" classname="com.instaone.datadriven.Customer_DataDriven"/>
  <testcase name="sameno" time="1.743" classname="com.instaone.datadriven.Customer_DataDriven"/>
  <testcase name="update" time="24.653" classname="com.instaone.datadriven.Customer_DataDriven"/>
</testsuite> <!-- com.instaone.datadriven.Customer_DataDriven -->
